.github/workflows/ci.yml

```diff
@@ -68,17 +73,20 @@ jobs:
         # see https://github.com/actions/setup-python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9
+          python-version: ${{ env.PYTHON_VERISON_DEFAULT }}
           architecture: 'x64'
       - name: Install poetry
         # see https://github.com/marketplace/actions/setup-poetry
         uses: Gr1N/setup-poetry@v7
         with:
-          poetry-version: 1.1.8
+          poetry-version: ${{ env.POETRY_VERSION }}
+      - name: get poetry virtualenvs
+        id: poetry-virtualenvs
+        run: echo "::set-output name=path::$(poetry config virtualenvs.path)"
```

Comment:
this way even windows setups will have poper caching 
see https://github.com/CycloneDX/cyclonedx-python/pull/261/files#r750775813